src/PostgREST/Error.hs

```diff
@@ -295,6 +289,9 @@ instance JSON.ToJSON ApiRequestError where
     Nothing
     (JSON.String <$> tableNotFoundHint schemaName relName tbls)
 
+  toJSON (NotImplemented details) = toJsonPgrstError
+    ApiRequestErrorCode25 "Feature not implemented" (Just $ JSON.String details) Nothing
+
```

Comment:
How about document this code on the errors page? Additionally, the NotImplemented error code should always remain in case we need it later.